styles.css

```
@@ -1,3 +1,138 @@
-body {
-    background-color: aquamarine;
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Mulish:ital,wght@0,200..1000;1,200..1000&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+* {
+  font-family: "Montserrat", sans-serif;
+  background-color: rgb(33, 26, 26);
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin: 20px 0 20px 10px;
+  padding: 20px 0 0 50px;
+}
+.main{
+    margin: 20px 0 20px 50px;
+}
+  .site-name {
```

```suggestion
.site-name {
```
Be careful with the indentation.  From this line, everything is one tab too much to the right.

---

hello @vdaguenet , 

Thank you for your valuable review and suggestions.
I have implemented 2 includes a function for sorting the movie array by one or a few selected properties. Now It  is  react to user inputs .

Thank you